circuit_maintenance_parser/parsers/equinix.py

```diff
@@ -54,18 +54,29 @@ def _parse_b(self, b_elements, data):
             impact (Status object): impact of the maintenance notification (used in the parse table function to assign an impact for each circuit).
         """
         impact = None
+        start_year = 0
+        end_year = 0
         for b_elem in b_elements:
+            if "SPAN:" in b_elem.text:
```

Comment:
why `.text` is needed here and not in the other if statements?

Reply:
the relevant text is in the text of the b element rather than next to the b element:

`<b>SPAN: 02-JUL-2021 - 03-JUL-2021</b>`

vs

`<b>UTC:</b> FRIDAY, 02 JUL 10:00 - FRIDAY, 02 JUL 15:00`